sql/src/test/java/io/crate/lucene/CommonQueryBuilderTest.java

```diff
@@ -423,7 +423,7 @@ public void testIpRange() throws Exception {
     public void testAnyEqOnTimestampArrayColumn() {
         assertThat(
             convert("1129224512000 = ANY(ts_array)").toString(),
-            is("ts_array:[1129224512000 TO 1129224512000]")
+            is("any_=(1129224512000, to_long_array(Ref{doc.users.ts_array, timestamp_array}))")
```

Comment:
There is something wrong with the cast semantics here. Literals should have preference when casting over columns. so this should end up casting `1129224512000` to `timestamp` to choose the fast query instead of the slower `any_=` functionf allback

Reply:
This must be a `FuncParams` thing :(